src/treeChooser/treeChooser.controller.js

```diff
@@ -552,7 +574,15 @@ function TreeChooserController(
      * Update exclusions on filter text change
      */
     $scope.$watch('vm.filterText', function () {
+      var active = vm.findActive();
```

Comment:
There's already logic that sets active on filter text. Determine where that is and modify that.